ruby/src/api.cc

```diff
@@ -238,7 +238,7 @@ namespace leatherman { namespace ruby {
     {
         v = rb_funcall(v, rb_intern("to_s"), 0);
         v = rb_str_encode(v, utf8_value("UTF-8"), 0, _nil);
-        size_t size = static_cast<size_t>(rb_num2ulong(rb_funcall(v, rb_intern("bytesize"), 0)));
+        size_t size = static_cast<size_t>(rb_num2ull(rb_funcall(v, rb_intern("bytesize"), 0)));
```

Comment:
Actually, I think I'm misreading Ruby's source. rb_num2ll and rb_num2ull always return 64-bit types. I lean towards always using that, even on 32-bit systems... mostly because the signature of rb_num2long changed and I don't want to have to support multiple function signatures.